client/utils.go

```diff
@@ -76,3 +77,24 @@ func ReadPageRequest(flagSet *pflag.FlagSet) (*query.PageRequest, error) {
 func NewClientFromNode(nodeURI string) (*rpchttp.HTTP, error) {
 	return rpchttp.New(nodeURI)
 }
+
+// WithPageKeyDecoded returns the provided flagSet with the page-key value base64 decoded (if it exists).
+// This is for when the page-key is provided as a base64 string (e.g. from the CLI).
+// ReadPageRequest expects it to be the raw bytes.
+//
+// Common usage: pageReq, err := client.ReadPageRequest(client.WithPageKeyDecoded(cmd.Flags()))
+func WithPageKeyDecoded(flagSet *pflag.FlagSet) *pflag.FlagSet {
```

Comment:
I changed the name to `FlagSetWithPageKeyDecoded` and made it also return an error. I then made `MustFlagSetWithPageKeyDecoded` so that the 1-line option is still there.